cmd/scale.go

```diff
@@ -353,12 +352,15 @@ func (sc *scaleCmd) run(cmd *cobra.Command, args []string) error {
 	}
 	var apiVersion string
 	sc.containerService, apiVersion, err = apiloader.LoadContainerServiceFromFile(sc.apiModelPath, false, true, nil)
+	if err != nil {
```

Comment:
So actually doing something with this `err` makes perfect sense. @amanohar @dmitsh could you take a look at this code and make sure that there isn't a reason why we actually *want* to absorb this error no matter what? (in which case we should throw away the `err` reference and replace with `_`).

I mean, we definitely *want* to not ignore errors, but if doing so in this scale code actually breaks something holistically, we want to know about it so that we can fix the entire stack.